docs/docs/reference/crd-reference/evaluationdefinition.md

```diff
@@ -26,6 +26,7 @@ spec:
 ```
 
 ## Fields
+<!-- markdownlint-disable MD007 -->
```

Comment:
you always need re-enable the rule  at the end of the section you want to ignore it for

Reply:
fixed